### PR TITLE
smartctl: add page

### PR DESCRIPTION
**Pull request by sommd**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/smartctl.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- View SMART health summary:
+
+`sudo smartctl --health {{/dev/<device>}}`
+
+- View device information:
+
+`sudo smartctl --info {{/dev/<device>}}`
+
+- Begin a (short) self-test:
```
Comment (agnivade) [inline pages/common/smartctl.md]: You can take short out of brackets and remove the tokens around "short" in the example. That way, the example directly matches the description.

File: pages/common/smartctl.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# smartctl
+
+> View a disk's [SMART](https://en.wikipedia.org/wiki/S.M.A.R.T.) data and other information.
```
Comment (agnivade) [inline pages/common/smartctl.md]: This is a console tool. So `[xyz](link)` will be displayed as is. I suggest you keep the wikipedia link in a second line.

File: pages/common/smartctl.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- View SMART health summary:
+
+`sudo smartctl --health {{/dev/<device>}}`
```
Comment (agnivade) [inline pages/common/smartctl.md]: For filesystems, we usually give a concrete example- like `/dev/sda1`. It makes it more clear. Please do that for all examples.

**Comment by agnivade** [comment]
Thanks for the PR ! This is a very important command. I gave some minor comments that needs some work.

File: pages/common/smartctl.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- View SMART health summary:
+
+`sudo smartctl --health {{/dev/sda}}`
+
+- View device information:
+
+`sudo smartctl --info {{/dev/sda}}`
+
+- Begin a short self-test:
```
Comment (sbrl) [inline pages/common/smartctl.md]: So this doesn't actually output the result of the short self-test?

Comment (sommd) [reply pages/common/smartctl.md]: Self tests run in the background, this command exits after starting the test.

There are tests that run in the foreground, but that pauses all other disk operations, which isn't a good idea if testing a mounted drive.

Check the results is shown in the next commands.

Comment (sbrl) [reply pages/common/smartctl.md]: Ah, ok. That's ok then - just wanted to check :-)

**Comment by sbrl** [comment]
Thanks, @sommd 😺

**Pull request merged**